### PR TITLE
Use fix-permissions for /etc/passwd instead of chmod g+w

### PR DESCRIPTION
**Pull request by smlx**
These are just some minor patches I made to the images while working on them recently.

### CODE REVIEW

File: images/commons/Dockerfile [hunk 1]
```diff
@@ -26,8 +26,6 @@ RUN apk update \
     && mkdir -p /lagoon/crontabs && fix-permissions /lagoon/crontabs \
     && ln -s /home/.bashrc /home/.profile
 
-RUN chmod g+w /etc/passwd
```
Comment (tobybellwood) [inline images/commons/Dockerfile:29]: do we not do this here because we assume no-one runs commons, only inherits, or is there another reason?

Comment (smlx) [reply images/commons/Dockerfile:29]: yes exactly. I don't think we run commons anywhere, do we?

Comment (tobybellwood) [reply images/commons/Dockerfile:29]: is there any harm in keeping this here (or using it's own fix-permissions?) - on the off-chance a derivative image doesn't fix permissions?

Comment (smlx) [reply images/commons/Dockerfile:29]: no, you're right there's no harm in leaving this here. I'll update the PR :+1:

**Review by tobybellwood — APPROVED** [review]

**Pull request merged**